src/main/kotlin/uk/gov/justice/digital/hmpps/manageoffencesapi/resource/AdminController.kt

```diff
@@ -60,6 +62,17 @@ class AdminController(
     return adminService.deactivateNomisOffence(offenceIds)
   }
 
+  @PostMapping(value = ["/nomis/offences/encouragement/{offenceId}"])
+  @PreAuthorize("hasRole('ROLE_NOMIS_OFFENCE_ACTIVATOR')")
+  @Operation(
+    summary = "Create encouragement offence for parent offence",
+    description = "Encouragement offence creates a new record with existing parent offence value, but with 'E' suffix to the offence code",
+  )
+  fun createEncouragementOffence(@PathVariable offenceId: Long): Offence {
```

Comment:
can we rename this variable to parentOffenceId rather than offenceId (all refs)?